doc/source/development/building_from_source.rst

```diff
@@ -108,6 +108,12 @@ for the shared lib, *e.g.* ``set (GDAL_LIB_OUTPUT_NAME gdal_x64 CACHE STRING ""
 
 Refer to :ref:`using_gdal_in_cmake` for how to use GDAL in a CMake project.
 
+To uninstall GDAL, from the build directory:
```

Comment:
maybe add this word of caution?
```suggestion
To uninstall GDAL, from the build directory (be careful: this does not restore the files that pre-existed to the last installation, but just removes the last installed files!)
```

Reply:
@rouault Does that mean that the previously installed GDAL may get broken and people should "ideally" reinstall it from packages?

Reply:
> Does that mean that the previously installed GDAL may get broken and people should "ideally" reinstall it from packages?

if people installed their custom GDAL in the same directory that the system one (they should not !), yes it will be broken

Reply:
OK! That probably explains why mine is still working; I blindy follow the instructions in gdal docs and it landed in /usr/local.
Thanks.